components/common/AsyncData.js

```diff
@@ -0,0 +1,38 @@
+
+class AsyncData {
+
+    static setTrackScreen = async (status) => {
+        saveData('TrackScreen', status)
+    }
+
+    static setTrackScreen = async () => {
+        return retrieveData('TrackScreen')
+    }
+
+    saveData = async (itemKey, status) => {
+        try {
+          await AsyncStorage.setItem(
+            itemKey,
+            status,
+          );
+        } catch (error) {
+          // Error saving data
+        }
+      };
+    
+    retrieveData = async (itemKey) => {
+        try {
+          const value = await AsyncStorage.getItem(itemKey);
+          if (value !== null) {
+            // We have data!!
+            console.log(value);
```

Comment:
does this need to be removed?

Reply:
Already removed in previous commits. However, thank you for bringing this to my attention.